composables/api/useLoginUser.ts

```diff
@@ -0,0 +1,60 @@
+import { ref } from 'vue'
+import { useRuntimeConfig, useState } from '#app'
+import { LoginRequest, User } from '@/lib/api/data-contracts'
+import { Users } from '@/lib/api/Users'
+
+interface LoginUserResponse {
+  loading: boolean
+  error: string | null
+}
+
+export const useLoginUser = (requestData: LoginRequest): LoginUserResponse => {
+  const responseLoading = ref(true)
+  const responseError = ref<string | null>(null)
+  const authToken = useState<string>('session_token')
```

Comment:
Do we not need a default value here? Does it get set to `''` or something?

Reply:
From what I've seen in the [documentation for the `useState` composable](https://v3.nuxtjs.org/guide/features/state-management), the initialization function isn't required, so I would guess that it initializes as `undefined` unless it was set elsewhere in the application's lifecycle. For the initial usage of this here, I think that should be fine, but I'm also fine setting a default if you or anyone else feels strongly about it.

Reply:
Ah okay. Yeah I'm fine with this. Resolving.